Base/usr/share/man/man7/Help-index.md

```diff
@@ -2,20 +2,21 @@
 
 ## Welcome to the SerenityOS on-line help system!
```

Comment:
now iʼm wondering what “on-line” means here exactly… 🤔

Reply:
This isn't something I've changed, it's the same wording from the current `Help-index.md` (the main page in the Help application).

The likely reason the author chose "on-line" will be because it's [the 90's way of saying "online"](https://grammarist.com/spelling/online-on-line/), which fits in with the [1990s aesthetic of SerenityOS](https://en.wikipedia.org/wiki/SerenityOS#Features).